python/ray/autoscaler/autoscaler.py

```diff
@@ -35,6 +35,8 @@
 RAY_SCHEMA_PATH = os.path.join(
     os.path.dirname(ray.autoscaler.__file__), "ray-schema.json")
 
+NODE_TYPE_CONFIG_KEYS = {"workers": "worker_nodes", "head": "head_node"}
```

Comment:
FYI I'm fixing this in https://github.com/ray-project/ray/pull/8632/commits/81190562830ee90aafa073a9a1a5701c68941421

to avoid creating new "workers" and "head" constants that aren't used elsewhere.